src/trezor-keyring.ts

```diff
@@ -410,8 +483,8 @@ class TrezorKeyring extends EventEmitter {
       domain = {},
       primaryType,
       // snake_case since Trezor uses Protobuf naming conventions here
-      domain_separator_hash, // eslint-disable-line camelcase
-      message_hash, // eslint-disable-line camelcase
+      domain_separator_hash, // eslint-disable-line
```

Comment:
Nit: It'd be nice to know the rule that we are disabling, so I'm wondering whether we should leave it in the comment.